07_yaml/test/test_yaml.rb

```diff
@@ -35,5 +35,5 @@ def test_listar_dados
     assert_equal({ nome: 'Bob', tipo_animal: 'cachorro', castrado: true }, pets_carregados[1])
 
     assert_equal 'Princesa', pets_carregados.first[:nome]
-   end
+    end
```

Comment:
[Not blocked]

Acredito que adicionou espaço sem querer.
```suggestion
   end
```